rplugin/python3/deoplete/deoplete.py

```diff
@@ -234,15 +234,14 @@ def profile_start(self, name):
             if self.__profile_flag:
                 return self.profile_start(name)
         elif self.__profile_flag:
-            self.__vim.command(
-                'echomsg \'profile start: {0}\''.format(name))
+            self.debug('Profile Start: {0}'.format(name))
             self.__profile_start = time.clock()
 
     def profile_end(self, name):
         if self.__profile_start:
-            self.__vim.command(
-                'echomsg \'profile end  : {0}, time={1}\''.format(
-                    name, time.clock() - self.__profile_start))
+            self.debug('Profile End  : {0:<25} {1}'.format(
```

Comment:
Maybe the Formatter should be set to use the newer format strings: https://docs.python.org/3.4/library/logging.html#logging.Formatter

So this could be simplified to:
```python
self.debug('Profile End  : {0:<25} time={:2.10f}', name, time.clock() - self.__profile_start)
```